src/main/java/com/sap/oss/phosphor/fosstars/model/value/ArtifactVersionValue.java

```diff
@@ -0,0 +1,69 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import java.util.Objects;
+
+/**
+ * A value for a feature that holds versions for an artifact.
+ */
+public class ArtifactVersionValue extends AbstractValue<ArtifactVersion> {
+
+  /**
+   * Versions of an artifact.
+   */
+  private final ArtifactVersion artifactVersion;
+
+  /**
+   * Initializes a value for a feature.
+   *
+   * @param feature The feature.
+   * @param artifactVersion The versions.
+   */
+  @JsonCreator
+  public ArtifactVersionValue(
+      @JsonProperty("feature") Feature<ArtifactVersion> feature,
+      @JsonProperty("version") ArtifactVersion artifactVersion) {
+
+    super(feature);
+    this.artifactVersion = artifactVersion;
+  }
+
+  @Override
+  public boolean isUnknown() {
+    return artifactVersion == null;
+  }
```

Comment:
I'll make `AbstractValue.isUnknown()` final. It has to return true. You can use `UnknownValue` to create unknown values.

```suggestion
```